app/Http/Middleware/VoteVerify.php

```diff
@@ -32,8 +33,9 @@ public function handle($request, Closure $next)
                     }
                     if (Auth::check()) {
                         $userId = $request->user()->id;
-                        $ids = explode("|", $vote->voted_user);
-                        if (!in_array($userId, $ids)) {
+                        // $ids = explode("|", $vote->voted_user);
```

Comment:
just delete it if it is considered deprecated 
[check this out](http://coolshell.cn/articles/4758.html)